habitat-lab/habitat/tasks/rearrange/actions/actions.py

```diff
@@ -455,6 +457,61 @@ def step(self, *args, **kwargs):
             self.update_base()
 
 
+@registry.register_task_action
+class BaseVelLegAnimationAction(BaseVelAction):
```

Comment:
@jimmytyyang I assume we eventually want Spot leg animation in habitat-lab `main`/`SIRo` for use in ML training. So eventually this change should go into those branches rather than live only here in `SIRo_hitl`. We can tackle this cleanup in October.